Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -0,0 +1,176 @@
+using Content.Server.Popups;
+using Content.Server.Cargo.Systems;
+using Content.Server.Cargo.Components;
+using Content.Server.Radio.EntitySystems;
+using Content.Server.Shipyard.Components;
+using Content.Server.Shuttles.Components;
+using Content.Server.Shuttles.Systems;
+using Content.Server.Station.Systems;
+using Content.Shared.GameTicking;
+using Content.Shared.Shipyard.Events;
+using Content.Shared.Shipyard.BUI;
+using Content.Shared.Shipyard.Prototypes;
+using Content.Shared.Shipyard.Components;
+using Content.Shared.Access.Systems;
+using Content.Shared.Access.Components;
+using Content.Shared.Shipyard;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+using Robust.Shared.Players;
+using Robust.Shared.Prototypes;
+using Content.Shared.Radio;
+
+namespace Content.Server.Shipyard.Systems
+{
```

Comment:
Should also be using file-scoped namespaces for new classes

Reply:
i dont understand? should the namespace be Content.Server.Shipyard.Systems.ShipyardConsole or similar?